recipes/raylib/all/conanfile.py

```diff
@@ -17,13 +18,19 @@ class RaylibConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "build_examples": [True, False],
+        "use_external_glfw": ["OFF","ON","IF_POSSIBLE"],
+        "opengl_version":["OFF","3.3","2.1","1.1","ES 2.0"]
```

Comment:
same, we can use `None` instead of string "OFF" here to indicate no OpenGL version